src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $color-light-gray;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin: 0 48px 46px 0;
```

Comment:
do not set external geometry to the bem blocks

Reply:
fix it 
![image](https://user-images.githubusercontent.com/98764641/204584684-1c3607d7-d785-409f-8c65-009a75a8cd13.png)